content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,117 @@
 ---
 title: "WebRTC"
-description: "WebRTC is a new specification that uses QUIC to offer an alternative to WebSocket. Conceptually, it can be considered WebSocket over QUIC.Learn about WebTransport and how it is used in libp2p."
+description: "WebRTC is a protocol for real-time communication and is used to establish connections between browsers and other nodes."
 weight: 110
 aliases:
     - "/concepts/transports/webrtc"
 ---
 
-Coming soon!
+## What is WebRTC?
+
+[WebRTC (Web Real-Time Communications)](https://webrtc.org/) is a framework for real-time
+communication and is used to establish browser-to-server and browser-to-browser connections
+between applications. WebRTC serves as a good choice for applications that need built-in support
+for media communication and do not have specific requirements for the underlying transport
+protocol.
+
+## WebRTC in libp2p
+
+### Key features
+
+- Peer connections: WebRTC enables [direct peer-to-peer connections](https://webrtc.org/getting-started/peer-connections)
+  between browsers and other nodes.
+
+- Data channel: WebRTC also provides peer-to-peer data channels called
+  [WebRTC data channels](https://developer.mozilla.org/en-US/docs/Games/Techniques/WebRTC_data_channels),
+  which works on
+  [SCTP (Stream Control Transmission Protocol)](https://en.wikipedia.org/wiki/Stream_Control_Transmission_Protocol).
+  A WebRTC data channel allows applications to send text or binary data over an active connection to a peer.
+
+- [NAT traversal](../nat/overview): WebRTC includes mechanisms (like
+  [ICE](https://datatracker.ietf.org/doc/rfc5245/))to connect to nodes that run behind
+  NATs and firewalls. In non-decentralized WebRTC, this can be facilitated by a
+  [TURN server.](https://webrtc.org/getting-started/turn-server),
+  but other signaling channels, such as WebSocket running on a central server, can also be used.
+  It is also possible to use a custom signaling protocol or a different signaling service.
+  Overall, this allows for faster and more efficient communication.
+
+- Security: WebRTC connections are encrypted using
+  [DTLS](https://en.wikipedia.org/wiki/Datagram_Transport_Layer_Security).
```

Comment:
Not sure we need a list here. I'd just say that we use WebRTC to connect from browsers to other nodes. Then point out that we don't use any of the multimedia features, but ONLY datachannels.